Dockerfile.google_cloud_sdk

```diff
@@ -0,0 +1,18 @@
+# using digestID of the 3rd party dockerhub image, as updates in the images might break the build
+FROM python@sha256:90834dba6381dfc3957573dc7a3e6c5c8ed255cf60079329a6da2b5e6d4257b8
```

Comment:
no, the reason being is, after some time, due to some updates in the official docker hub image, the build was failing. so we decided to move to download and use as like we use now. this will help us to modify the local tests more easily